edx_proctoring/api.py

```diff
@@ -1424,7 +1424,7 @@ def _get_timed_exam_view(exam, context, exam_id, user_id, course_id):
     attempt_status = attempt['status'] if attempt else None
     has_due_date = True if exam['due_date'] is not None else False
     if not attempt_status:
-        if has_due_date_passed(exam['due_date']):
+        if has_due_date_passed(exam['due_date']) and not _is_self_paced_course(user_id, course_id):
```

Comment:
My question to this approach is: in the proctoring service, do we want to always persist the `due date` information passed in from the `context` object? Basically, the true source of `due date` information is always outside of the proctoring service? 
I think I am definitely in support of the above idea because I do believe the exam information, since it's part of the course anyways, should be passed into the proctoring service as the trusted data.